src/deepsparse/pipeline.py

```diff
@@ -617,17 +619,29 @@ def _run_with_dynamic_batch(
         ]
         # wait for all inferences to complete before joining outputs
         concurrent.futures.wait(futures)
-        outputs = [future.result() for future in futures]
-        return self.output_schema.join(outputs)
+        collected_results = [future.result() for future in futures]
+        outputs, inference_timing_batch = zip(*collected_results)
+        return self.output_schema.join(outputs), InferenceTimingSchema.aggregate(
```

Comment:
Wondering, whether we should include the inference timing object inside the output schema. i.e create field `InferenceTiming` inside every `OutputSchema`